app/src/main/java/at/roteskreuz/stopcorona/model/entities/exposure/DbSentTemporaryExposureKeys.kt

```diff
@@ -0,0 +1,39 @@
+package at.roteskreuz.stopcorona.model.entities.exposure
+
+import android.os.Parcelable
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import at.roteskreuz.stopcorona.skeleton.core.model.entities.DbEntity
+import kotlinx.android.parcel.Parcelize
+import java.util.*
+
+/**
+ * Describes a sent temporary exposure key through it's associated rolling start interval number
+ * and password.
+ */
+@Entity(
+    tableName = "sent_temporary_exposure_keys"
+)
+@Parcelize
+data class DbSentTemporaryExposureKeys(
+    @PrimaryKey
+    val rollingStartIntervalNumber: Int,
```

Comment:
Are you sure it will be unique?

Reply:
Based on my current understanding of the Exposure SDK this should be unique. All the exposure keys in a particular time frame should be associated with a `rollingStartIntervalNumber`